src/components/experiment-tracking/details/index.js

```diff
@@ -38,8 +44,19 @@ const Details = ({ selectedRuns, sidebarVisible }) => {
           'details-mainframe--sidebar-visible': sidebarVisible,
         })}
       >
-        <RunMetadata isSingleRun={isSingleRun} runs={runMetadata} />
-        <RunDataset isSingleRun={isSingleRun} trackingData={runTrackingData} />
+        <RunMetadata
+          isSingleRun={isSingleRun}
+          runs={runMetadata}
+          enableShowChanges={enableShowChanges}
+          pinnedRun={pinnedRun}
+          setPinnedRun={setPinnedRun}
```

Comment:
I really think we should just start sorting our props and attributes now, because if we're going to do it in the future anyways we're just adding to our technical debt.